contracts/token/ERC6909/draft-ERC6909.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IERC6909} from "../../interfaces/draft-IERC6909.sol";
+import {Context} from "../../utils/Context.sol";
+import {IERC165, ERC165} from "../../utils/introspection/ERC165.sol";
+
+contract ERC6909 is Context, ERC165, IERC6909 {
+    error ERC6909InsufficientBalance(address sender, uint256 balance, uint256 needed, uint256 id);
+    error ERC6909InsufficientAllowance(address spender, uint256 allowance, uint256 needed, uint256 id);
+    error ERC6909InvalidReceiver(address receiver);
+    error ERC6909InvalidSender(address sender);
+
+    mapping(uint256 id => mapping(address owner => uint256)) private _balances;
```

Comment:
Why not `mapping(address owner => mapping(uint256 id => uint256))`? I feel is more consistent

Reply:
This is following ERC1155 impl. Given that we've strayed pretty far from storage parity at this point its probably not necessary.